courses/machine_learning/images/mnist_models.ipynb

```diff
@@ -0,0 +1,1012 @@
+{
```

Comment:
<p>Nice!</p><p> </p><p>A few suggestions:</p><ul><li>substitute everywhere: nueron -&gt; neuron</li><li>I'd split create_and_train_Model into a function "build_model" that returns a compiled model and another one that only train_and_evaluate it</li><li>you don't need to return both the history and the model, since history has model as attribute I believe (history.model) </li><li>you are importing ModelCheckpoint but not using it</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/608/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Aw man, the neuron spelling gets me every time :)</p><p>I can get ModelCheckpoint to work locally, but not on ML Engine. Have you had any luck?</p>

Reply:
<p>Maybe we can just remove ModelCheckpoint from the import if it's not needed by any part of the code. </p>